src/together/resources/finetune.py

```diff
@@ -449,9 +484,13 @@ async def create(
             n_checkpoints (int, optional): Number of checkpoints to save during fine-tuning.
                 Defaults to 1.
             batch_size (int, optional): Batch size for fine-tuning. Defaults to max.
-            learning_rate (float, optional): Learning rate multiplier to use for training
-                Defaults to 0.00001.
+            min_lr_ratio (float, optional): Ending learning rate as a percentage of the initial learning rate for
+                learning rate scheduler. Defaults to 0.0.
+            min_lr_ratio (float, optional): Ending learning rate for learning rate scheduler.
+                Defaults to 0.0.
             warmup_ratio (float, optional): Warmup ratio for learning rate scheduler.
```

Comment:
I think you accidentally removed the learning rate argument from here. Also, the comments for `FineTuning.create` apply here

Reply:
Sorry, added back and updated.